go/pkg/pwagent/daemon.go

```diff
@@ -11,9 +14,14 @@ import (
 	"pathwar.land/go/pkg/pwapi"
 	"pathwar.land/go/pkg/pwcompose"
 	"pathwar.land/go/pkg/pwdb"
+	"pathwar.land/go/pkg/pwinit"
 )
 
-func Daemon(ctx context.Context, cli *client.Client, logger *zap.Logger) error {
+const (
+	defaultChallengesPath = "challenges/web"
```

Comment:
You should use an absolute path to allow running `Pathwar` commands from any directory

Cool default is `/tmp/pathwar-agent/`

This should be configurable via CLI flag

Reply:
Not needed anymore, it was only used for prepare